app/src/main/java/br/org/cesar/knot_setup_app/wrapper/LogWrapper.java

```diff
@@ -0,0 +1,28 @@
+package br.org.cesar.knot_setup_app.wrapper;
+import android.content.Context;
+import android.util.Log;
+
+import br.org.cesar.knot_setup_app.BuildConfig;
+
+public class LogWrapper {
+
+    private static String DEV_LOG = "DEV-LOG";
```

Comment:
I suggest you use the **final** modifier on this variable declaration if you do not want it to be modified. Additionally, I suggest changing DEV_LOG name to something more clear.